src/Commands/DrupalCommands.php

```diff
@@ -282,8 +282,20 @@ public function settingsSetup(array $options = [
       'root' => InputOption::VALUE_REQUIRED,
     ])
     {
+        $settings_file = $options['root'].'/sites/default/settings.php';
+        $settings_local_file = $options['root'] . '/sites/default/' . $this->getConfig()->get('drupal.settings_local_file');
         return $this->collectionBuilder()->addTaskList([
-            $this->taskAppendConfiguration($options['root'].'/sites/default/default.settings.php', $this->getConfig())->setConfigKey('drupal.settings'),
+            $this->taskFilesystemStack()->copy($options['root'] . '/sites/default/default.settings.php', $settings_file),
```

Comment:
We need to set the third parameter `$force` to `TRUE` otherwise the runner will not override an already existing `settings.php`.